src/components/dynamic/admin/services/contacts/Contact.jsx

```diff
@@ -5,6 +5,7 @@ import Select from "@/components/dynamic/Select";
 import { useState } from "react";
 import { api } from "@/utils/api";
 import toaster from "@/utils/toaster";
+import { useSearchParams, usePathname, useRouter } from "next/navigation";
```

Comment:
it should use server side params https://nextjs.org/docs/app/api-reference/file-conventions/page#searchparams-optional